src/utils/stark/stark-key.ts

```diff
@@ -92,10 +93,6 @@ function getStarkPublicKeyWithXCoordinate(keyPair: ec.KeyPair): string {
   return encUtils.sanitizeHex(getXCoordinate(getPublic(keyPair, true)));
 }
 
-export function getKeyPairFromPrivateKey(privateKey: string): ec.KeyPair {
```

Comment:
Why do we want to remove this function, and is there any replacement?

Reply:
What is this function for and where is it used?

Reply:
@Kostia Kim 
WalletSdk needs the getKeyPairFromPrivateKey as it requires initialing the **base signer** to sign the message.  I am not sure if there’s any other way to do so.
So for Wallet Sdk, we store the key pair as a string in local storage. But when users do the **signMessage**, we need to have a **keypair object to sign**. So I don’t know how to get the keypair object without that function. 
please provide the alternative

@vmiguez-immutable @RenatoChitolina

Reply:
@Kastet looking through the code, and I find we have `getKeyPair` function which seems to have the same implementation of `getKeyPairFromPrivateKey`,we may use it as an alternative if it will remain exists. 🤔 
So  can you confirm that we don't have a plan to remove/change that function?

Reply:
We want to remove or at least private most of the utility functions.